website/docs/cloud-docs/api-docs/notification-configurations.mdx

```diff
@@ -51,7 +51,7 @@ Notifications are sent as the run progresses, and can be triggered on one or mor
 | Created            | `"run:created"`         | A run is created and enters the [Pending stage](/cloud-docs/run/states#1-the-pending-stage)                                                    |
 | Planning           | `"run:planning"`        | A run acquires the lock and starts to execute.                                                      |
 | Needs Attention    | `"run:needs_attention"` | A plan has changes and Terraform requires user input to continue. This may include approving the plan or a [policy override](/cloud-docs/run/states#7-the-policy-check-stage). |
-| Applying           | `"run:applying"`        | A run enters the [Apply stage](/cloud-docs/run/states#8-the-apply-stage), where Terraform makes the infrastructure changes described in the plan.                            |
+| Applying           | `"run:applying"`        | A run enters the [Apply stage](/cloud-docs/run/states#9-the-apply-stage), where Terraform makes the infrastructure changes described in the plan.                            |
 | Completed          | `"run:completed"`       | A run has completed successfully.                                     |
 | Errored            | `"run:errored"`         | A run has terminated early due to error or cancellation.                                          |
 | Drifted            | `"assessment:drifted"`  | Terraform Cloud detected configuration drift. This option is only available if drift detection is enabled for the workspace.  Not available in Terraform Enterprise.                                                   |
```

Comment:
```suggestion
| Completed          | `"run:completed"`       | A run completes successfully.                                     |
| Errored            | `"run:errored"`         | A run terminates early due to error or cancellation.                                          |
| Drifted            | `"assessment:drifted"`  | Terraform Cloud detected configuration drift. This option is only available if you enabled drift detection for the workspace.  Not available in Terraform Enterprise.                                                   |
```

Suggest changing "has completed" to "completes", "has terminated" to "terminates", and "drifted detection is enabled" to "you enabled drift detection" to avoid passive voice style issues.